src/login/logind-session-device.c

```diff
@@ -104,6 +105,19 @@ static void sd_eviocrevoke(int fd) {
         }
 }
 
+static void sd_hidiocrevoke(int fd) {
+        static bool warned = false;
+
+        assert(fd >= 0);
+
+        if (ioctl(fd, HIDIOCREVOKE, NULL) < 0) {
+                if (errno == EINVAL && !warned) {
+                        log_warning_errno(errno, "Kernel does not support hidraw-revocation: %m");
```

Comment:
The cached value should be used to avoid retrying if this means the kernel cannot support the operation

Reply:
fixed, thanks